programs/ambassadors/ambassadors-program.md

```diff
@@ -0,0 +1,62 @@
+This document covers the organizational and operational aspects of the JSON Schema Ambassador Program.
+
+### What is the JSON Schema Ambassador Program?
+
+The JSON Schema Ambassador Program aims to bring JSON Schema closer to both current users and potential new project users. It also aims to provide users and community members with the necessary tools and resources to:
+
+- Promote collaboration among the JSON Schema users base.
+- Engage with the project in some way, either as a contributor, blogger, speaker, etc.
+- Speak at community events or write technical content (i.e., blog posts) that solves community needs.
+- Nurture community growth and a welcoming environment to spread knowledge about the project.
+
+### How does the program work?
+
+- The ambassador selection process will be as transparent as possible.
+- The model currently in place for adding members to the Technical Steering Committee through PR will be used.
+- The TSC or delegated maintainers of community repositories will evaluate applications according to compliance requirements via a traditional PR flow.
+- All ambassador information will be displayed on the JSON Schema website in a list after the PR is merged.
+
+### Ambassador requirements
+
+The participation will be determined and checked annually. The minimum requirement for being an ambassador is to make **4 contributions per year** in the form of articles, talks, videos, podcasts, presentations, driving initiatives, etc.
+
+Ambassadors can be invited to review content (i.e., articles, talks, videos) related to the community. These reviews are optional and will be counted as an additional contribution.
+
+Some clarifications on the type of contributions:
+    
+- Articles, videos, and podcasts can be published on [JSON Schema.com/blog](https://www.JSON Schema.com/blog) or other domains. 
+- Talks and presentations can be displayed at different events or workshops. Within it, JSON Schema should appear at the core of the presentation. The presentation would never be used as a disguised marketing tool for other products/projects.
+- Special contributions should be driven to support the community or enhance its visibility (i.e., getting new sponsors or driving initiatives) 
+- If we did not identify the contributions, just let us know.
+         
+Examples: 
+
+1. If contributors publish 3 articles and propel 1 initiative about JSON Schema in eight months, they will become ambassadors for the whole year.
+2. If contributors make 1 presentation, write 2 articles, and propel 1 initiative, they will become ambassadors for the whole year.
+3. If you were accepted as ambassador on the 5th of May 2022, you have already fulfilled the requirements. On the 5th of May 2023, we will check if you qualify again after the 5th of May 2022.
+   
+### Ambassadors duties
+
+- Be in tune with JSON Schema's mission and values.
+- Always respect the [code of conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Have we considered hosting this on the website so it has a nicer URL?

Reply:
Great idea! I just filled a new issue for this enhancement. https://github.com/json-schema-org/website/issues/278